### PR TITLE
Note Addition

### PR DESCRIPTION
**Pull request by joinimran**
As external inbound emails cannot be encrypted using a mail flow rule and OME. 

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/594#issuecomment-554560409

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ You can define mail flow rules for triggering message encryption with the new OM
 
 1. In a web browser, using a work or school account that has been granted global administrator permissions, [sign in to Office 365](https://support.office.com/article/b9582171-fd1f-4284-9846-bdd72bb28426#ID0EAABAAA=Web_browser).
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by joinimran** [comment]
Copy review suggestions applied.

**Comment by KCCross** [comment]
Instead of including this note, I added the following language to the introduction in the article:

You can't encrypt inbound mail from senders outside of your organization.

**Pull request closed without merging**